app/org/sagebionetworks/bridge/hibernate/HibernateAccount.java

```diff
@@ -0,0 +1,251 @@
+     * identifying info).
+     */
+    @Id
+    public String getId() {
+        return id;
+    }
+
+    /** @see #getId */
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /** Study ID the account lives in. */
+    public String getStudyId() {
+        return studyId;
+    }
+
+    /** @see #getStudyId */
+    public void setStudyId(String studyId) {
+        this.studyId = studyId;
+    }
+
+    /** Account email address. */
+    public String getEmail() {
+        return email;
+    }
+
+    /** @see #getEmail */
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    /** Map of custom account attributes. Never returns null. */
+    @CollectionTable(name = "AccountAttributes", joinColumns = @JoinColumn(name = "accountId",
+            referencedColumnName = "id"))
+    @Column(name = "attributeValue")
+    @ElementCollection(fetch = FetchType.EAGER)
+    @MapKeyColumn(name = "attributeKey")
+    public Map<String, String> getAttributes() {
+        if (attributes == null) {
+            attributes = new HashMap<>();
+        }
+        return attributes;
+    }
+
+    /** @see #getAttributes */
+    public void setAttributes(Map<String, String> attributes) {
+        // Note: Hibernate doesn't support copying this into a separate map.
+        this.attributes = attributes;
+    }
+
+    /** Map of consents, keyed by a composite of subpopulation ID and signedOn. Never returns null. */
+    @CollectionTable(name = "AccountConsents", joinColumns = @JoinColumn(name = "accountId",
+            referencedColumnName = "id"))
+    @ElementCollection(fetch = FetchType.EAGER)
+    @MapKeyClass(HibernateAccountConsentKey.class)
+    public Map<HibernateAccountConsentKey, HibernateAccountConsent> getConsents() {
+        if (consents == null) {
+            consents = new HashMap<>();
+        }
+        return consents;
+    }
+
+    /** @see #getConsents */
+    public void setConsents(Map<HibernateAccountConsentKey, HibernateAccountConsent> consents) {
+        // Note: Hibernate doesn't support copying this into a separate map.
+        this.consents = consents;
+    }
+
+    /** Epoch milliseconds when the account was created. */
+    public Long getCreatedOn() {
+        return createdOn;
+    }
+
+    /** @see #getCreatedOn */
+    public void setCreatedOn(Long createdOn) {
+        this.createdOn = createdOn;
+    }
+
+    /** Account health code. */
+    public String getHealthCode() {
+        return healthCode;
+    }
+
+    /** @see #getHealthCode */
+    public void setHealthCode(String healthCode) {
+        this.healthCode = healthCode;
+    }
+
+    /** Account health ID, which maps to health code. */
+    public String getHealthId() {
+        return healthId;
+    }
+
+    /** @see #getHealthId */
+    public void setHealthId(String healthId) {
+        this.healthId = healthId;
+    }
```

Comment:
In migration are we going to get rid of healthId and just work directly with healthCode? Is the accounts table encrypted?